README.md

```diff
@@ -26,7 +26,7 @@ No more plaintext credentials in your home directory. Automatically authenticate
 <p align="center">
   <picture>
   <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/45081667/227191964-9629476d-a49e-475d-b8cb-2115c302025d.gif">
-   <img src="https://user-images.githubusercontent.com/45081667/229802932-d3a5de19-82f1-43bf-825d-befcc649ed29.gif" alt="Example of 1Password Shell Plugins with AWS: user runs an `aws` command, a Touch ID prompt shows up, and `aws` is automatically authenticated" style="max-width: 100%; display: inline-block;" />
+   <img src="https://user-images.githubusercontent.com/45081667/232083691-2bee9a7e-4ec5-4e30-966f-3511cbbd2142.gif" alt="Example of 1Password Shell Plugins with AWS: user runs an `aws` command, a Touch ID prompt shows up, and `aws` is automatically authenticated" style="max-width: 100%; display: inline-block;" />
```

Comment:
This image adds a gray line at the bottom:

<img width="914" alt="image" src="https://user-images.githubusercontent.com/7430639/232084998-032ce580-d7d5-4152-a025-c356ad1ec50b.png">

Reply:
The new addition removes that line